app/experimenter/legacy-ui/core/package.json

```diff
@@ -41,7 +41,7 @@
     "react": "16.13.1",
     "react-bootstrap": "1.3.0",
     "react-dom": "16.13.1",
-    "react-select": "3.1.0"
+    "react-select": "^3.1.1"
   },
```

Comment:
just curious, is there context of why the version bump/changes for react-select? I remember something about react-select hell in slack....

Reply:
Well I think this whole workspace thing is kinda wonky where you define the requirements per 'project', ie for core, rapid, visualization, nimbus-ui, but they share a lock file and they all get installed into one node modules folder, and I think they weirdly overlap somehow?  So when I removed the rapid ui, suddenly nimbus didn't have access to the @types/react-select, so I think that was installed for rapid but not nimbus-ui but nimbus-ui could still import it as if it were, so i just reinstalled it everywhere and that semed to clear it up.